owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/authentication/OCAuthenticationViewModel.kt

```diff
@@ -62,7 +62,7 @@ class OCAuthenticationViewModel(
     fun loginBasic(
         username: String,
         password: String,
-        updateAccountIfAlreadyExists: Boolean = false
+        updateAccountWithUsername: String?
```

Comment:
I would rename this parameter to `updateUserAccountName` for all the classes using it , what do you think?

Reply:
We want to update the account that matches with that account name, so i think that the naming is not bad at all.